Doctrine/Phpcr/StaticContent.php

```diff
@@ -11,14 +11,13 @@
 
 namespace Symfony\Cmf\Bundle\ContentBundle\Doctrine\Phpcr;
 
-use Doctrine\ODM\PHPCR\HierarchyInterface;
 use PHPCR\NodeInterface;
 use Symfony\Cmf\Bundle\ContentBundle\Model\StaticContent as ModelStaticContent;
 
 /**
  * PHPCR specific static content.
  */
-class StaticContent extends ModelStaticContent implements HierarchyInterface
```

Comment:
> though there are likely conflicts then with other dependencies?

Yeah, we'll need to go full blown dev stability now in all CMF packages (as we need to require SonataDoctrinePhpcrAdminBundle 2.0-dev)